tox.ini

```diff
@@ -76,7 +76,7 @@ deps =
     behave==1.2.4
     behave-pytest==0.1.1
 commands =
-    python -m behave {posargs}
+    behave {posargs}
```

Comment:
there was a reason that this was done (though I don't remember what)...howcome you're changing it?